src/components/MoveAdder.ts

```diff
@@ -0,0 +1,13 @@
+import { IMove } from "battlemovr";
+import { GeneralComponent } from "gamestartr";
+
+import { FullScreenPokemon } from "../FullScreenPokemon";
+import { IPokemon } from "./Battles";
+
+export class MoveAdder<TGameStartr extends FullScreenPokemon> extends GeneralComponent<TGameStartr> {
+
+    //returns whether the move has been successfully added or not
```

Comment:
Use proper `/** */` "JSDoc" comments like in other files on this class and this method.

Reply:
See other methods that use `@returns` too